servus/CMakeLists.txt

```diff
@@ -17,7 +17,6 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 set(SERVUS_PUBLIC_HEADERS
-  ${COMMON_INCLUDES}
```

Comment:
I didn't quite get this in the CMAKE PR. These files are now added automatically to the sources by common_library?